src/pyrenew/metaclass.py

```diff
@@ -126,6 +126,38 @@ def _assert_sample_and_rtype(
     return None
 
 
+def compute_incidence_observed_with_delay(
+    incidence_to_observation_rate: ArrayLike,
```

Comment:
I think `ratio` is more clear than `rate`.

Reply:
I disagree because of the x/y versus y/x problem. If you say "x_y_ratio", the baseline assumption is x/y. Here we typically represent things in units of observations per incidence (e.g. probability of hospital admissions given infection). If you call it "`ncidence_to_observation_ratio`, that implies incidence per observation, i.e. the reciprocal.

Reply:
will refrain from renaming based on @dylanhmorris's comment above.

Reply:
I think `incidence_to_observation_rate` is easily confused with the concept of a reporting delay (the rate referring to the time to reporting rather than the proportion of reporting).

What about `ascertainment_ratio`, `reporting_ratio`, or `underreporting_ratio`? I'd also be fine with `ascertainment_rate`, `reporting_rate`, or `underreporting_rate`.

Reply:
I can change it to `ascertainment rate` to avoid possible confusion with `hospitalization_reporting_rate`.

It seems `rate/ratio` is used interchangeably in the codebase (e.g `latentadmissions.py`), we should probably change that for consistency.

Reply:
Yes, can you make an issue for making this consistent?

Proposal: since these are Bayesian models, why don't we avoid the potential confusions associated with rates/ratios altogether and use conditional probability language:

`p_<x>_given_<y>`

Reply:
created #410

Reply:
Thanks!